### PR TITLE
Prevent null pointer when exporting file credentials

### PR DESCRIPTION
**Pull request by timja**
Requires https://github.com/jenkinsci/credentials-plugin/pull/135
Fixes https://github.com/jenkinsci/configuration-as-code-plugin/issues/1170

### Your checklist for this pull request

🚨 Please review the [guidelines for contributing](../blob/master/docs/CONTRIBUTING.md) to this repository.

- [ ] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side) and not your master branch!
- [ ] Ensure that the pull request title represents the desired changelog entry
- [ ] Please describe what you did
- [ ] Link to relevant issues in GitHub or in [Jenkins JIRA](https://issues.jenkins-ci.org)
- [ ] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Did you provide a test-case? That demonstrates feature works or fixes the issue.

### CODE REVIEW

File: plugin/src/main/java/io/jenkins/plugins/casc/impl/configurators/DataBoundConfigurator.java [hunk 1]
```diff
@@ -279,21 +279,24 @@ public CNode describe(T instance, ConfigurationContext context) throws Exception
...
-                    args[i] = converted;
-                } else if (Set.class.isAssignableFrom(p.getType())) {
-                    args[i] = Collections.singleton(converted);
-                } else {
-                    args[i] = Collections.singletonList(converted);
+            if (a != null) {
+                Object value = a.getValue(instance);
+                if (value != null) {
+                    Object converted = Stapler.CONVERT_UTILS.convert(value, a.getType());
+                    if (converted instanceof Collection || p.getType().isArray() || !a
```
Comment (casz) [inline plugin/src/main/java/io/jenkins/plugins/casc/impl/configurators/DataBoundConfigurator.java]: This needs to be formatted differently it looks odd.

**Comment by casz** [comment]
Build fails due to upper bound deps.

**Comment by timja** [comment]
sorted

**Comment by timja** [comment]
(this previously passed, re-triggering)

**Comment by timja** [comment]
I've been waiting for https://github.com/jenkinsci/credentials-plugin/pull/135 to be merged and released as we're currently depending on an incremental version in the integration tests 

cc @jvz

**Comment by jvz** [comment]
I've published credentials 2.3.2 @timja

**Comment by casz** [comment]
@jvz I think you forgot to merge the PR: https://github.com/jenkinsci/credentials-plugin/pull/135 :scream

**Comment by jvz** [comment]
Womp womp. Publishing 2.3.3 right now as well as an updated changelog in it.

**Comment by casz** [comment]
👏

File: integrations/src/test/java/io/jenkins/plugins/casc/CredentialsTest.java [hunk 2]
```diff
@@ -67,6 +67,31 @@ public void testDomainScopedCredentials() {
         assertEquals("secret", creds.get(0).getPassword().getPlainText());
     }
 
+    @Test
```
Comment (jglick) [inline integrations/src/test/java/io/jenkins/plugins/casc/CredentialsTest.java]: Add

```java
@Issue("JENKINS-60467")
```

IIUC

**Pull request merged**